keyboards/kyria/keymaps/mattir/config.h

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2019 Thomas Baart <thomas@splitkb.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#ifdef OLED_DRIVER_ENABLE
+  #define OLED_DISPLAY_128X64
+  #define OLED_TIMEOUT 300000
+#endif
+
+#ifdef RGBLIGHT_ENABLE
+  #define RGBLIGHT_LIMIT_VAL 150
+  #define RGBLIGHT_ANIMATIONS
+  #define RGBLIGHT_HUE_STEP 8
+  #define RGBLIGHT_SAT_STEP 8
+  #define RGBLIGHT_VAL_STEP 8
+  #define RGBLIGHT_SLEEP
+#endif
+
+// If you are using an Elite C rev3 on the slave side, uncomment the lines below:
+#define SPLIT_USB_DETECT
+#define NO_USB_STARTUP_CHECK
```

Comment:
Are you using the 3.0 elite C?  if not, you don't want/need this. 

```suggestion
// #define SPLIT_USB_DETECT
// #define NO_USB_STARTUP_CHECK
```

Reply:
Yes, I'm using an Elite-C on each half.

Reply:
Are they the 3.0 elite C, specifically?  If not, then you don't need this.  If so, then you're good to go.

Reply:
These are Elite-C low profile rev 4. So they're not specifically v 3.0, but they are 3.0+, which I suspect would still qualify as needing this specified?

Reply:
Then you don't need these.  Only the 3.0 had the flaw that needs the split usb detect.  The v4 has the correct hardware. 

If you want a good writeup on the issue, you can see it here: https://medium.com/@keebio/the-case-of-the-wayward-elite-c-73f0fd691f88

Reply:
Ah, very cool. I will remove the lines and test locally on my keeb, just to be safe and push a new commit after testing.